javascripts/projectsService.js

```diff
@@ -132,27 +133,27 @@ define(['underscore'], _ => {
     return _.flatten(results, (arr1, arr2) => arr1.append(arr2));
   };
 
-  var TagBuilder = function() {
-    var _tagsMap = {},
+  const TagBuilder = function() {
+    const _tagsMap = {},
```

Comment:
Let's split these two up so they can be assigned separately:

```suggestion
    const _tagsMap = {};
```